doc/developer-notes.md

```diff
@@ -83,7 +83,7 @@ code.
     separate words (snake_case).
     - Class member variables have a `m_` prefix.
     - Global variables have a `g_` prefix.
-  - Compile-time constant names are all uppercase, and use `_` to separate words.
+  - Constant names are all uppercase, and use `_` to separate words.
```

Comment:
Now that you removed the "compile-time" requirement, it makes it unclear what happens to non-global constants, such as `const auto CURRENT_TIME = GetTime<std::chrono::seconds>();`

Reply:
@MarcoFalke Imagine a global constant `const auto int BLA = ComplexFunction();`, that's a constant variable, but it's not a compile-time nor link-time constant (neither compiler or linked knows its value).

Reply:
I know, and my point is that these (not a compile-time nor link-time constant) haven't been UPPER_CASED in the past and that this pull is changing the rule here.

Just asking to clarify.

Reply:
Gotcha.

Reply:
Oh, you're right. It wasn't my intention to change any rule, just to clarify the language. Will fix.